### PR TITLE
Implement Swagger API Documentation with docker run

### PR DESCRIPTION
**Pull request by Cyantosh0**
What has been done:
* API specs for user routes added
* Add swagger-ui container to test APIs locally
Run app and Browse to http://localhost:80 to test APIs

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,5 +37,19 @@ services:
...
+        - "80:8080"
+    volumes:
+        - ./docs:/usr/share/nginx/html/docs
+    environment:
+        API_URL: docs/openapi.yaml
+        # For multiple specifications
+        # URLS: "[{ url: \"docs/first_openapi.yaml\", name: \"FirstApiSpec\" },
+        #         { url: \"docs/second_openapi.yaml\", name: \"SecondApiSpec\" },
+        #     ]"
+
```
Comment (dipeshdulal) [inline docker-compose.yml]: @Cyantosh0 
Let's read port mapping from the environment variable as done in other services. Let's remove the comment in the environment as well.

**Review by dipeshdulal — CHANGES_REQUESTED** [review]
@Cyantosh0 Let's add readme

File: docker-compose.yml (hunk 1 above)
Comment (dipeshdulal) [reply docker-compose.yml]: Other things look fine to me. Thank You. 🙇

Comment (Cyantosh0) [reply docker-compose.yml]: @dipeshdulal sir, I have updated things as per your request. Please let me know, if there's anything.  🙇

**Review by dipeshdulal — APPROVED** [review]

**Pull request merged**